cmd/vclusterctl/cmd/find/find.go

```diff
@@ -137,6 +137,21 @@ func findInContext(context, name, namespace string, timeout time.Duration) ([]VC
 				continue
 			}
 
+			var paused string
+
+			paused, _ = p.Annotations[constants.PausedAnnotation]
```

Comment:
I guess we should check here if p.Annotations is not nil as otherwise this might cause a nil pointer panic (although probably very rarely)